### PR TITLE
Add/setup db seeding/#66

### PR DESCRIPTION
**Pull request by Ghaith-Chrit**
This PR closes: #66 

Please reset your database before testing this script. It is best to drop the database and then apply the provided migration.

### CODE REVIEW

File: Server/ConcordiaCurriculumManager/Program.cs [hunk 1]
```diff
@@ -69,14 +69,9 @@ public static void Main(string[] args)
         builder.Services.AddDbContext<CCMDbContext>(options =>
         {
             var dataSourceBuilder = new NpgsqlDataSourceBuilder(dbSetting!.ConnectionString);
-            dataSourceBuilder.MapEnum<RoleEnum>();
```
Comment (JamesPartsafas) [inline Server/ConcordiaCurriculumManager/Program.cs:72]: Out of curiosity, why did you remove this line?

**Review by JamesPartsafas — APPROVED** [review]
Looks good. It's easy to extend, and very nice use of generics.

File: Server/ConcordiaCurriculumManager/Program.cs (hunk 1 above)
Comment (Ghaith-Chrit) [reply Server/ConcordiaCurriculumManager/Program.cs:72]: This was mainly used to create a database enum similar to the RoleEnum. Thus, when you view the table in the database, you won't see integers but the role as an actual string (actually an enum value). I removed it since the other enums were not added in the same way, resulting in some discrepancies. Also, it is easier to seed an integer than an enum in the seeder script.
